docs/_sources/more_examples.rst.txt

```diff
@@ -80,6 +82,21 @@ This behavior can be altered in two different ways:
    :language: python
    :emphasize-lines: 36
 
+Accessing and Raising an Exception
+----------------------------------
+
+While it's possible to halt the graph execution due to unhandled exceptions
+(see `Halting Graph Execution upon Exceptions`_),
+these exceptions are not raised from within the :func:`~async_graph_data_flow.AsyncExecutor.execute` call.
+Instead, :class:`~async_graph_data_flow.AsyncExecutor`'s :attr:`~async_graph_data_flow.AsyncExecutor.exceptions`
+allows access to the exceptions from the nodes,
+and you can determine what to do with this information
+(e.g., raise an exception on your own).
+
+.. literalinclude:: ../../examples/raising_an_exception.py
+   :language: python
+   :emphasize-lines: 24-27
```

Comment:
In this pull request, I've re-generated the Sphinx doc HTML pages. I've verified that this new section looks nice and correctly rendered:

![Screenshot 2023-03-02 at 12 57 49 PM](https://user-images.githubusercontent.com/86482098/222515729-ab94f8e7-367c-495e-bc54-bcae1a2654d3.png)